### ISSUE TITLE
[Bug]: SAML app is unable to be deactivated

### ISSUE THREAD

**Issue by bitfactory-sem-denbroeder** [zitadel/zitadel#5571]
### Preflight Checklist

- [X] I could not find a solution in the documentation, the existing issues or discussions
- [X] I have joined the [ZITADEL chat](https://zitadel.com/chat)

### Environment

Self-hosted

### Describe the bug

We integrated AWS with zitadel SSO, however upon creating (and successfully linking the 2 together) we saw that disabling the app in the interface did not mean logging in fails. AWS goes to zitadel and zitadel authorizes my request.

In short: disabling a SAML app does not disable it.

### To reproduce

1. Create a zitadel integration with AWS
2. Disable said AWS SAML app in zitadel
3. Login to AWS

### Expected behavior

I expected to not be able to login to AWS

### Version

v2.22.2

### Operating System

K8s

**Comment by fforootd** [comment]
Hm this sounds really like a bug any idea on this @livio-a @stebenz

**Comment by bitfactory-sem-denbroeder** [comment]
I followed the steps in the documentation: https://github.com/zitadel/zitadel/blob/main/docs/docs/guides/integrate/services/aws-saml.md
